modules/katello_upload.py

```diff
@@ -152,15 +155,24 @@ def main():
     entity_dict['product'] = find_product(module, name=entity_dict['product'], organization=entity_dict['organization'])
     entity_dict['repository'] = find_repository(module, name=entity_dict['repository'], product=entity_dict['product'])
 
-    package = False
+    content_unit = False
```

Comment:
Wouldn't it be easier, more consistent to set it to `None` here?
`None` evaluates to `False` anyway. So a simple `if not content_unit:` does the job below.